test/SlowTests/Client/Attachments/AttachmentsReplication.cs

```diff
@@ -1564,136 +1564,111 @@ public async Task AttachmentWithSameStreamSameNameAndDifferentContentTypeShouldB
         [Fact]
         public async Task ResolvedToLatestOnAttachmentConflictShouldRemoveDuplicateAttachment()
         {
-            using (var store1 = GetDocumentStore())
-            using (var store2 = GetDocumentStore())
+            using var store1 = GetDocumentStore();
+            using var store2 = GetDocumentStore();
```

Comment:
This single change is causing wall of pink. Please revert this and avoid doing this kind of refactoring in future. I'm unable to track changes here.

Reply:
In this case, I actually rearrange the whole test.
If I will separate this in a different commit?

Reply:
Sure